src/cmds/qsub.c

```diff
@@ -465,23 +465,24 @@ comma_token(char *str)
 char *
 expand_varlist(char *varlist)
 {
-	char  *v_value1 = NULL;
-	char *v_value2 = NULL;
-	char *vn = NULL;
-	char *vv = NULL;
-	char *p1, *p2, *p;
-	char *ev;
-	int  v_value1_sz=0;
-	char *pc;
-	int special_char_cnt = 0;
-	int len = 0;
+	char	*v_value1 = NULL;
+	char	*v_value2 = NULL;
+	char	*vn = NULL;
+	char	*vv = NULL;
+	char	*p1, *p2, *p;
+	char	*ev;
+	int	v_value1_sz=0;
+	char	*pc;
+	int	special_char_cnt = 0;
+	int	len = 0;
 
 	/*
 	 * count special characters as they are escaped with '\' in copy_env_value function
-	 * so that this is useful while calculating the accurate size of the destination string
+	 * so that this is useful while calculating the accurate size of the destination string.
+	 * Also calculating the length of the string.
 	 */
 	pc = varlist;
-	for(;*pc;pc++) {
+	for (;*pc;pc++) {
```

Comment:
@suresh-thelkar i think @prakashcv13 wanted you to add spaces around the ";" and not after the for

Reply:
Done. 
@prakashcv13 / @subhasisb 
Please look into the latest commit for changes and provide your sign off/further review comments if any.